### PR TITLE
[IMP] inventory: add manual valuation section

### PR DESCRIPTION
**Pull request by Felicious**
Task ID [#2453902](https://www.odoo.com/web#id=2453902&cids=3&menu_id=4720&action=333&active_id=3835&model=project.task&view_type=form)

### Summary
Rewrite of [Using inventory valuation](https://www.odoo.com/documentation/14.0/applications/inventory_and_mrp/inventory/management/reporting/using_inventory_valuation.html) that:
1. Adds section explaining **Manual Valuation** button in the Inventory Valuation reports by product
2. Add images relating to the functionality
3. Removes 1st and 2nd pronouns from examples
4. Made summaries and point of the document more concise and clear
5. Rename headers to be action-focused
6. Add references to other pieces of internal documentation
7. Include clearer configs and how-tos to explain product screenshots

**Task**: Request from Product Owner to [add section explaining the manual valuation button](https://www.odoo.com/web#id=2453902&menu_id=4720&cids=3&action=333&active_id=3835&model=project.task&view_type=form)

### Update to PR
3/29: 
- Add reference tag to link to external doc, *Inventory valuation configuration*. 
- Also changed the heading to sentence case

### CODE REVIEW

**Comment by StraubCreative** [comment]
@samueljlieber can you review this ASAP since it's prio PR for a PO?

File: content/applications/inventory_and_mrp/inventory/management/reporting/using_inventory_valuation.rst [hunk 1]
```diff
@@ -1,77 +1,107 @@
...
 
-I can also easily access the accounting entry that has been generated
-(in case of automated inventory valuation).
+In Odoo, automatic inventory valuation records are also recorded in the :guilabel:`Accounting` app.
+To access these accounting entries,navigate to :menuselection:`Accounting --> Journal Entries`, and
+look for `Inventory Valuation` entries with the `STJ` prefix in the :guilabel:`Journal` and
+:guilabel:`Number` columns, respectively.
+
+For this example, search the :guilabel:`Reference` column for the :guilabel:`Receipt` `WH/IN/00001`
+to select the record for the inventory valuation of 10 tables. Below is the accounting entry.
```
Comment (samueljlieber) [inline content/applications/inventory_and_mrp/inventory/management/reporting/using_inventory_valuation.rst:61]: a couple points to make here:

1. we can remove the :guilabel: & capitalization from _receipt_
2. "Below is the accounting entry." This text relies on the image to provide context, when the text should really provide all of the needed context even without an image. Lets reword this to follow this principle: 
> For this example, search the :guilabel:`Reference` column for the receipt `WH/IN/00001` to select the record for the inventory valuation of 10 tables and view the accounting entry.

File: content/applications/inventory_and_mrp/inventory/management/reporting/using_inventory_valuation.rst [hunk 2]
```diff
@@ -1,77 +1,107 @@
...
 
 .. image:: using_inventory_valuation/use_inventory_valuation_04.png
     :align: center
```
Comment (samueljlieber) [inline content/applications/inventory_and_mrp/inventory/management/reporting/using_inventory_valuation.rst:65]: filename, max-width, and alt tag

File: content/applications/inventory_and_mrp/inventory/management/reporting/using_inventory_valuation.rst [hunk 3]
```diff
@@ -1,77 +1,107 @@
...
 
 Deliver a product
 -----------------
 
-In the same logic, when a table will be delivered, the stock valuation
-will be impacted and you will have access to a similar information.
+In the same logic, when a table is shipped to a customer and leaves the warehouse, the stock
+valuation decreases. The :guilabel:`Valuation` smart button on the :abbr:`DO(Delivery Order)` displays the stock valuation record.
```
Comment (samueljlieber) [inline content/applications/inventory_and_mrp/inventory/management/reporting/using_inventory_valuation.rst:70]: add a space in the `:abbr:` tag

File: content/applications/inventory_and_mrp/inventory/management/reporting/using_inventory_valuation.rst [hunk 4]
```diff
@@ -1,77 +1,107 @@
...
 
 .. image:: using_inventory_valuation/use_inventory_valuation_05.png
     :align: center
```
Comment (samueljlieber) [inline content/applications/inventory_and_mrp/inventory/management/reporting/using_inventory_valuation.rst:73]: filename, max-width, and alt tag

File: content/applications/inventory_and_mrp/inventory/management/reporting/using_inventory_valuation.rst [hunk 5]
```diff
@@ -1,77 +1,107 @@
...
 The inventory valuation report
 ------------------------------
 
-The summary of this is accessible via the inventory valuation report
-(accessible from :menuselection:`Inventory --> Reporting --> Inventory Valuation`). It gives
-you, product per product, the value of your stock. By clicking on the
-button *Inventory At Date*, you can have the same information for a
-past date.
+To view the current value of all products in the warehouse, go to :menuselection:`Reporting -->
+Inventory Valuation` in the :guilabel:`Inventory` app. The records are organized by product,
```
Comment (samueljlieber) [inline content/applications/inventory_and_mrp/inventory/management/reporting/using_inventory_valuation.rst:79]: swap :guilabel:Inventory fir :menuselection:Inventory here, this is because we are directing the user to the Inventory app, so a :menuselection: is used

File: content/applications/inventory_and_mrp/inventory/management/reporting/using_inventory_valuation.rst [hunk 6]
```diff
@@ -1,77 +1,107 @@
...
+and selecting the drop-down displays detailed records with the date, quantity, and value of the
+shipment.
```
Comment (samueljlieber) [inline content/applications/inventory_and_mrp/inventory/management/reporting/using_inventory_valuation.rst:81]: The headers in the UI Date, Quantity, and Total Value should have :guilabel:s because we are referring to them on the UI in this sentence.

Also the UI shows Total Value, not just Value.

File: content/applications/inventory_and_mrp/inventory/management/reporting/using_inventory_valuation.rst [hunk 7]
```diff
@@ -1,77 +1,107 @@
...
 
 .. image:: using_inventory_valuation/use_inventory_valuation_06.png
     :align: center
```
Comment (samueljlieber) [inline content/applications/inventory_and_mrp/inventory/management/reporting/using_inventory_valuation.rst:84]: filename, max-width, and alt tag

File: content/applications/inventory_and_mrp/inventory/management/reporting/using_inventory_valuation.rst [hunk 8]
```diff
@@ -1,77 +1,107 @@
...
+
+.. note::
+    :guilabel:`Inventory At Date` shows the inventory valuation of products available during a past
+    date.
```
Comment (samueljlieber) [inline content/applications/inventory_and_mrp/inventory/management/reporting/using_inventory_valuation.rst:88]: admonition text alignment

File: content/applications/inventory_and_mrp/inventory/management/reporting/using_inventory_valuation.rst [hunk 9]
```diff
@@ -1,77 +1,107 @@
...
+
+Update product unit price
+~~~~~~~~~~~~~~~~~~~~~~~~~
```
Comment (samueljlieber) [inline content/applications/inventory_and_mrp/inventory/management/reporting/using_inventory_valuation.rst:91]: empty line after headings

File: content/applications/inventory_and_mrp/inventory/management/reporting/using_inventory_valuation.rst [hunk 10]
```diff
@@ -1,77 +1,107 @@
...
+For any company, the lead time, risk of supply chain failures, and other factors contribute to
+invisible costs. Although Odoo attempts to accurately represent the stock value, *manual valuation*
+serves as an additional tool to update the unit price of products.
```
Comment (samueljlieber) [inline content/applications/inventory_and_mrp/inventory/management/reporting/using_inventory_valuation.rst:94]: lets capitalize _Manual Valuation_ here because that is how it is displayed on the UI and we are introducing the term for the first time

File: content/applications/inventory_and_mrp/inventory/management/reporting/using_inventory_valuation.rst [hunk 11]
```diff
@@ -1,77 +1,107 @@
...
+
+This operation is intended for products that can be purchased and received for a cost greater than
+0, or have product categories set with :guilabel:`Costing Method` set as either `Average Cost
+(AVCO)` or `First In First Out (FIFO)`.
+
+.. image:: using_inventory_valuation/use_inventory_valuation_07.png
+    :align: center
```
Comment (samueljlieber) [inline content/applications/inventory_and_mrp/inventory/management/reporting/using_inventory_valuation.rst:101]: filename, max-width, and alt tag

The information this screenshot is providing could be better received by the user if we cut out much of the unnecessary context. Its very rare we need to show the entire UI in a screenshot when referring to a single element.

Lets crop this down to just the necessary information and then add a highlight/arrow for the actionable elements.

File: content/applications/inventory_and_mrp/inventory/management/reporting/using_inventory_valuation.rst [hunk 12]
```diff
@@ -1,77 +1,107 @@
...
+
+The Product Revaluation form updates the calculation of inventory valuation by increasing or
+decreasing the unit price of each product.
+
+.. image:: using_inventory_valuation/use_inventory_valuation_08.png
+    :align: center
```
Comment (samueljlieber) [inline content/applications/inventory_and_mrp/inventory/management/reporting/using_inventory_valuation.rst:107]: filename, max-width, and alt tag

File: content/applications/inventory_and_mrp/inventory/management/reporting/using_inventory_valuation.rst (hunk 11 above)
Comment (samueljlieber) [reply content/applications/inventory_and_mrp/inventory/management/reporting/using_inventory_valuation.rst:101]: add-manual-valuation
VS
image

File: content/applications/inventory_and_mrp/inventory/management/reporting/using_inventory_valuation.rst [hunk 13]
```diff
@@ -1,77 +1,107 @@
...
+serves as an additional tool to update the unit price of products.
+
+This operation is intended for products that can be purchased and received for a cost greater than
+0, or have product categories set with :guilabel:`Costing Method` set as either `Average Cost
+(AVCO)` or `First In First Out (FIFO)`.
+
+.. image:: using_inventory_valuation/use_inventory_valuation_07.png
+    :align: center
+
+The Product Revaluation form updates the calculation of inventory valuation by increasing or
```
Comment (samueljlieber) [inline content/applications/inventory_and_mrp/inventory/management/reporting/using_inventory_valuation.rst:103]: Lets :guilabel:Product Revaluation because we are referring to it directly in this sentence

File: content/applications/inventory_and_mrp/inventory/management/reporting/inventory_valuation_config.rst [hunk 14]
```diff
@@ -1,7 +1,9 @@
 =================================
-Inventory Valuation Configuration
+Inventory valuation configuration
 =================================
 
+.. _inventory/inventory_valuation_config:
+
```
Comment (samueljlieber) [inline content/applications/inventory_and_mrp/inventory/management/reporting/inventory_valuation_config.rst:6]: Since we are editing this doc by adding an internal reference, we may as well improve it while we are here 🙂

See the diff (files changed) for the technical improvements I made here.. I cant comment line by line 😁

**Review by samueljlieber — CHANGES_REQUESTED** [review]
Hi @Felicious 👋 nice job getting this PR started! I've gone through and made some technical changes as suggestions below for your reference. Please read through them and let me know if you approve of the changes. 

I will be pushing these technical changes up as a second commit on this PR so you can see exactly what I changes in the **Files changed** tab. Once I recieve your approval, I will squash the commits, rebase, and push back up for a final review.

Here is a summary of the changes I made:
- only break lines once they have reached 100 characters
- alignment in admonition blocks, seealso, and image attributes
- always empty line after a heading
- images should always have an `:alt:` tag
- images should not be larger than 768px wide
- images should focus on the important elements and not show unnecessary context. Dont be afraid to crop and markup images
- images should always be secondary and complimentary to written instructions, don't use images as part of an instruction
- doc path references should be global (starting at the `applications/...` folder)

Overall these are all points that will become more familiar as you work on the User Docs, so keep it up 💪 awaiting your approval on these changes 🙂

File: content/applications/inventory_and_mrp/inventory/management/reporting/using_inventory_valuation.rst (hunk 13 above)
Comment (StraubCreative) [reply content/applications/inventory_and_mrp/inventory/management/reporting/using_inventory_valuation.rst]: Not sure this applies...the logic is correct but it's a gray area since it's not a heading, button, filter, label, etc.
I'll ask JCS

**Comment by samueljlieber** [comment]
Implemented technical changes and approving changes in https://github.com/odoo/documentation/commit/80d0151929a57fda3ad446aae8656f9f16b01db9, ready for final review @StraubCreative.

**Review by samueljlieber — APPROVED** [review]

**Review by StraubCreative — CHANGES_REQUESTED** [review]
Hi @Felicious 

Great job on this PR 👍 
Thank you too @samueljlieber for your help here 🙂 

My comments below mostly revolve around high-level wording suggestions that will really make your work shine. I found very few technical errors which is great!

My suggestions, generally speaking, revolve around:
- being _very_ specific around UI instruction/navigation or when introducing concepts for the first time.
- adding context clues throughout the documentation article, especially when starting new paragraphs or sections. Remind readers, _constantly_, where_ they are, what just happened, what is happening / about to happen, and why we're doing what we're doing (when it's relevant to do so).
- some other background on why we write the way we do in relationship to how doc readers typically consume this kind of content, and how we can support them in having a positive and educational experience.

Will push up all the change requests in a separate commit after this. 
- If you agree with the changes, let me know and I can go ahead and close this out. 
- If you disagree with anything or need further clarification, please detail your dissent on the individual CRs and we can figure it out.

TIA/LMK 🚀

**Review by Felicious — COMMENTED** [review]
Agree with all of your changes, Zach! Really liking how everything reads after your review. :thumbsup:

**Comment by StraubCreative** [comment]
Change requests implemented on [cd17645](https://github.com/odoo/documentation/pull/3948/commits/cd17645bfd3b24f6ef89b84d2921719f8b7881a7). All checks pass 👍 

Note: @Felicious will need to do a targeted PR for 16 to include revisions for that version. Please create a Task in Odoo Project and be sure to update this Task and tag PO that it's merged.

@robodoo r+
